packages/core/src/routes/interaction/actions/submit-interaction.ts

```diff
@@ -132,10 +181,19 @@ const parseNewUserProfile = async (
           user,
         }))
     ),
+    conditional(
+      connectorId &&
```

Comment:
📝 The condition for getting new blockchain and social profile is the same `connectorId`, should probably distinguish between them more